docs/docs/graphql/core/databases/postgres/schema/computed-fields.mdx

```diff
@@ -21,7 +21,8 @@ import TabItem from '@theme/TabItem';
 
 Computed fields are virtual values or objects that are dynamically
 computed and can be queried along with a table/view's columns. Computed
-fields are computed when requested for via [custom SQL functions](https://www.postgresql.org/docs/current/sql-createfunction.html)
+fields are computed upon request. They are computed by 
```

Comment:
It might be even better to say "upon API requests" but that feels like more of a judgement call, so I'll defer to reviewers about that. I just wanted to suggesting "upon request" rather than "when requested for", and splitting into two sentences.